flink-ml-lib/src/test/java/org/apache/flink/ml/classification/OnlineLogisticRegressionTest.java

```diff
@@ -0,0 +1,444 @@
+                        .fit(onlinePredictTable)
+                        .getModelData()[0];
+        List<Row> modelList = IteratorUtils.toList(tEnv.toDataStream(models).executeAndCollect());
+        assertEquals(10, modelList.size());
+    }
+
+    @Test
+    public void testFitWithInitLrModel() throws Exception {
+        Table initLrModel =
+                new LogisticRegression()
+                        .setFeaturesCol(FEATURE_COL)
+                        .setLabelCol(LABEL_COL)
+                        .fit(trainDenseTable)
+                        .getModelData()[0];
+        Table onlineTrainTable = getTable(50, 1000, TRAIN_DENSE_ROWS, 4, false);
+        Table models =
+                new OnlineLogisticRegression()
+                        .setFeaturesCol(FEATURE_COL)
+                        .setInitialModelData(initLrModel)
+                        .setGlobalBatchSize(100)
+                        .setLabelCol(LABEL_COL)
+                        .fit(onlineTrainTable)
+                        .getModelData()[0];
+        List<Row> modelList = IteratorUtils.toList(tEnv.toDataStream(models).executeAndCollect());
+        assertEquals(10, modelList.size());
+    }
+
+    @Test
+    public void testDenseFitAndPredict() throws Exception {
+        Table onlineTrainTable = getTable(2, 2000, TRAIN_DENSE_ROWS, 2, false);
+        Table onlinePredictTable = getTable(2, 3000, PREDICT_DENSE_ROWS, 2, false);
+        OnlineLogisticRegressionModel model =
+                new OnlineLogisticRegression()
+                        .setFeaturesCol(FEATURE_COL)
+                        .setInitialModelData(initDenseModel)
+                        .setGlobalBatchSize(500)
+                        .setLabelCol(LABEL_COL)
+                        .fit(onlineTrainTable);
+        Table resultTable = model.setPredictionCol(PREDICT_COL).transform(onlinePredictTable)[0];
+        verifyPredictionResult(resultTable, 4);
+    }
+
+    @Test
+    public void testSparseFitAndPredict() throws Exception {
+        double[] doubleArray = new double[] {0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1};
+        Table initSparseModel =
+                tEnv.fromDataStream(env.fromElements(Row.of(new DenseVector(doubleArray), 0L)));
+        Table onlineTrainTable = getTable(5, 2000, TRAIN_SPARSE_ROWS, 4, true);
+        Table onlinePredictTable = getTable(5, 3000, PREDICT_SPARSE_ROWS, 4, true);
+        OnlineLogisticRegressionModel model =
+                new OnlineLogisticRegression()
+                        .setFeaturesCol(FEATURE_COL)
+                        .setInitialModelData(initSparseModel)
+                        .setGlobalBatchSize(500)
+                        .setLabelCol(LABEL_COL)
+                        .fit(onlineTrainTable);
+        Table resultTable = model.setPredictionCol(PREDICT_COL).transform(onlinePredictTable)[0];
+        verifyPredictionResult(resultTable, 4);
+    }
+
+    @Test
+    public void testParam() {
```

Comment:
nits: could we order the methods in this class to be more consistent with test methods in other test classes, e.g. move the `testParam()` to be the first method in this class?